### PR TITLE
Changes how vaurca phoron leaking works

### PR DESCRIPTION
**Pull request by Alberyk**
-vaurca phoron tanks will only leak when damaged
-removed the breeder commanding power
-removes the vaurca monkey exhaling phoron

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm [hunk 1]
```diff
@@ -73,8 +73,7 @@
 	flags = NO_SCAN | NO_SLIP
 
 	inherent_verbs = list(
-		/mob/living/carbon/human/proc/bugbite,
-		/mob/living/carbon/human/proc/hivenet
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm]: Might wanna changelog this? Otherwise, looks good

Comment (MarinaGryphon) [reply code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm]: yeah this is a good, lore approved change but, changelog

File: html/changelogs/alberyk-vaurcathing.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+author: Alberyk
+
+delete-after: True
+
+changes: 
+  - tweak: "Vaurca phoron tanks will now only leak when damaged."
```
Comment (MarinaGryphon) [inline html/changelogs/alberyk-vaurcathing.yml]:
```suggestion
  - tweak: "Vaurca phoron tanks will now only leak when damaged."
  - tweak: "Ta consulars can no longer issue orders over hivenet or bite off heads."
```

Comment (MarinaGryphon) [inline html/changelogs/alberyk-vaurcathing.yml]:
```suggestion
  - tweak: "Vaurca phoron tanks will now only leak when damaged."
  - tweak: "Ta consulars can no longer issue orders over hivenet in any way besides actually saying them."
```

File: html/changelogs/alberyk-vaurcathing.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+author: Alberyk
```
Comment (MarinaGryphon) [inline html/changelogs/alberyk-vaurcathing.yml]:
```suggestion
author: Alberky
```

**Pull request merged**